### PR TITLE
docs update for --update-fstab

### PR DESCRIPTION
**Pull request by timfeirg**
as discussed in https://github.com/juicedata/juicefs/pull/2462, merge after 1.1 is release (and widely adopted), and then we can present `--update-fstab` as the recommended way to set up auto-mount

### CODE REVIEW

File: docs/zh_cn/getting-started/for_distributed.md [hunk 1]
```diff
@@ -135,21 +135,17 @@ JuiceFS 进程需要具有读写 `--cache-dir` 目录的权限。
...
-
-```shell
-sudo cp /usr/local/bin/juicefs /sbin/mount.juicefs
-```
-
-编辑 `/etc/fstab` 配置文件，遵照 fstab 的规则添加一条新记录：
-
-```
-redis://tom:mypassword@myjfs-sh-abc.redis.rds.aliyuncs.com:6379/1    /mnt/myjfs    juicefs    _netdev,max-uploads=50,writeback,cache-size=512000     0  0
+可以用 `juiefs mount --update-fstab` 直接设置出自动挂载，例如：
```
Comment (xiaogaozi) [inline docs/zh_cn/getting-started/for_distributed.md:138]: Need to prompt users that this is an unreleased feature and supported operating system. And the "Command Reference" documentation is missing a description of the `--update-fstab` option.

File: docs/zh_cn/guide/mount_at_boot.md [hunk 2]
```diff
@@ -6,21 +6,31 @@ slug: /mount_juicefs_at_boot_time
 
 # 启动时自动挂载 JuiceFS
 
-这是一篇关于如何在启动时自动挂载 JuiceFS 的指南。
+在确认挂载成功，可以正常使用以后，可以参考本节内容设置开机自动挂载。
 
 ## Linux
 
-拷贝 `juicefs` 为 `/sbin/mount.juicefs`，然后按照下面的格式添加一行到 `/etc/fstab`：
+挂载命令的 `--update-fstab` 参数能直接帮你做好设置：
```
Comment (xiaogaozi) [inline docs/zh_cn/guide/mount_at_boot.md:13]: Need to prompt users that this is an unreleased feature

**Review by xiaogaozi — CHANGES_REQUESTED** [review]

File: docs/zh_cn/guide/mount_at_boot.md (hunk 2 above)
Comment (timfeirg) [reply docs/zh_cn/guide/mount_at_boot.md:13]: this pr is to be merged after 1.1 release

Comment (xiaogaozi) [reply docs/zh_cn/guide/mount_at_boot.md:13]: Still need to prompt the user that this feature depends on a specific version

Comment (timfeirg) [reply docs/zh_cn/guide/mount_at_boot.md]: add notes on available version (v1.1.0)

**Review by xiaogaozi — APPROVED** [review]

**Comment by timfeirg** [comment]
After further discussion, this pr need to revert changes for tutorials, and add heads-up for version requirement

**Comment by timfeirg** [comment]
conflicts have been resolved.

**Pull request merged**